canarieapi/app_object.py

```diff
@@ -29,9 +24,10 @@
 APP.wsgi_app = ReverseProxied(APP.wsgi_app)
 
 # Config
-APP.config.from_object(default_configuration)
```

Comment:
In fact moving that inside the else is a breaking change : 
The parameter here : https://github.com/Ouranosinc/CanarieAPI/blob/master/canarieapi/default_configuration.py#L12
is not override by : https://github.com/bird-house/birdhouse-deploy/blob/5c5813677951247890f32ea07366c4ba75b61f5f/birdhouse/config/canarie-api/docker_configuration.py.template
So the default value was expected to be used. 
Sorry to have missed that before :(

Reply:
So we have to keep this line (32) where it is, which means we can't import default_configuration conditionally.

If I followed along correctly, we could then still use `from canarieapi import default_configuration` because we have `CANARIE_API_CONFIG_FN` or should I roll back to the modified cron command (path.insert) with `import default_configuration` ?

Reply:
You are right in both case.
We cannot import default_configuration conditionally.
And we should still be able to use the from canarieapi import default_configuration because it's ok to use the package installed one as we can override it by the CANARIE_API_CONFIG_FN.
In short, just move the line 31 and 32 above the if and we should be good!

Reply:
Not breaking: 
https://github.com/Ouranosinc/CanarieAPI/blob/0f28c609aabca3e79a0721e6b3215377148d139d/canarieapi/api.py#L128

Reply:
But true that it won't hurt to have the default loaded and then overridden by whatever (even partial) definition there was in `CANARIE_API_CONFIG_FN` file.

Reply:
Changes have been applied